cmd/state-installer/installer.go

```diff
@@ -156,6 +169,25 @@ func (i *Installer) sanitizeInput() error {
 	return nil
 }
 
+func (i *Installer) installSvcApp() (*app.App, error) {
```

Comment:
As discussed; it's a bit of a worry that we are generating this in the installer, rather than at build time. Not going to iterate on this at this time, but worth commenting in the code.